pkg/controller/plan/kubevirt.go

```diff
@@ -869,7 +832,7 @@ func (r *KubeVirt) GetGuestConversionPod(vm *plan.VMStatus) (pod *core.Pod, err
 
 // Delete the PVC consumer pod on the destination cluster.
 func (r *KubeVirt) DeletePVCConsumerPod(vm *plan.VMStatus) (err error) {
-	list, err := r.GetPodsWithLabels(r.consumerLabels(vm.Ref))
+	list, err := r.GetPodsWithLabels(r.consumerLabels(vm.Ref, true))
```

Comment:
it looks like we add all the labels, including the migration label, to the consume pod and we always drop the migration label when querying the consumer pods - why do we drop the migration label when querying?